quickstart.org

```diff
@@ -55,57 +82,97 @@ To buy:
    + Mozzarella
 #+END_SRC
 
-Lists can also contain checkboxes: ~[ ]~, ~[-]~, ~[X]~
+Lists can contain checkboxes ~[ ]~, ~[-]~, ~[X]~.
 #+BEGIN_SRC org
 - [ ] do
 - [-] the
 - [X] thing
 #+END_SRC
-* Images
-Just link to an image, and don't give a description. Seriously, that's it. \\
-~[[https://upload.wikimedia.org/wikipedia/commons/5/5d/Konigsberg_bridges.png]]~
 
-#+attr_html: :height 120px
-[[https://upload.wikimedia.org/wikipedia/commons/5/5d/Konigsberg_bridges.png]]
+Lists can contain tags (and checkboxes at the same time).
+#+BEGIN_SRC org
+- [ ] fruits :: get apples
+- [X] veggies :: get carrots
+#+END_SRC
+
 * Links
-Just put a target in between two square brackets, like so: ~[[target]]~. Want to
-add a description? Just add a ~[description]~ after the target, i.e.
-~[[target][desc]]~.
+To create a link put the target between two square brackets, like so: ~[[target]]~.
+To add a description add it in square brackets after the target ~[[target][desc]]~.
```

Comment:
If we could not have two "add"s so close to each other I think that could improve the flow of this sentence.
How about replacing the second one with "insert"?

Reply:
"To add a description include it in square ..."?

Reply:
Just the first alternative that came to mind for me, "To add a description insert it in square brackets after the target...".
Quite minor, but I'm not sure that "To **add** a ... **add** ..." flows that well.

Hmmm, maybe better to just reword the sentance and remove the 'it' as well, something like:
"You can insert a description after the target in square brackets".